simplefs/simplefs.go

```diff
@@ -520,8 +528,7 @@ func (k *SimpleFS) SimpleFSMakeOpid(_ context.Context) (keybase1.OpID, error) {
 	return opid, err
 }
 
-// SimpleFSClose - Close OpID, cancels any pending operation.
-// Must be called after list/copy/remove
```

Comment:
These comments are from the avdl and its which generated output, which we should update too. I'll make a ticket.